javascript/javascript1/week1/agefy.js

```diff
@@ -0,0 +1,7 @@
+const yearOfBirth = prompt("Enter year of Birth: ");
+const yearFuture = prompt("Enter the future year when you want your age?");
+const age = (yearFuture - yearOfBirth);
+console.log("Age-ify (A future age calculator):-");
+console.log("Your year of birth is :" + " " + yearOfBirth + " " + "AD");
+console.log("The year you want your age to be is : "+ " " + yearFuture + " " + "AD");
```

Comment:
You could also use backticks for interpolation here. It usually makes it more readable.
```javascript
console.log(`Your year of birth is : ${yearOfBirth} AD`);
console.log(`The year you want your age to be is : ${yearFuture} AD`);
```

And even if you decide not to, you don't need the spaces in their own quotation marks:
`"Your year of birth is : " + yearOfBirth + " AD"`

Reply:
yes , i will change it 
thanks